### PR TITLE
Add support for PNP over SSL

### PR DESCRIPTION
**Pull request by lpellegr**

### CODE REVIEW

**Comment by lpellegr** [comment]
Retest this please.

File: config/network/node.ini [hunk 1]
```diff
@@ -1,15 +1,26 @@
...
-### PNP Protocol configuration
+### PNP protocol configuration
 #proactive.pnp.port=1100
 
-### PAMR Protocol configuration
+### PNPS protocol configuration
+#proactive.pnps.authenticate=true
+#proactive.pnps.default_heartbeat=9000
+#proactive.pnps.idle_timeout=60000
+#proactive.pnps.keystore=/path/to/keystore.jks
```
Comment (fviale) [inline config/network/node.ini]: why not put the default path to the config/web/keystore ?

File: config/network/node.ini [hunk 2]
```diff
@@ -1,15 +1,26 @@
...
+#proactive.pnps.keystore.password=your-password
```
Comment (fviale) [inline config/network/node.ini]: default keystore password in config/web/keystore is activeeon

File: config/network/node.ini [hunk 3]
```diff
@@ -19,8 +30,9 @@ proactive.communication.protocol=pnp
 # on windows :
 #proactive.pamrssh.key_directory=c:\\Users\\login\\.ssh
 
-
 ### Common network configuration
+#proactive.net.interface=ethX
+#proactive.net.netmask=192.168.1.0/24
```
Comment (fviale) [inline config/network/node.ini]: good idea to add these suggestions

File: config/network/server.ini [hunk 4]
```diff
@@ -1,18 +1,35 @@
...
-### PNP Protocol configuration
+### PNP protocol configuration
 proactive.pnp.port=64738
 
-### PAMR Protocol configuration
+### PNPS protocol configuration
+#proactive.pnps.authenticate=true
+#proactive.pnps.default_heartbeat=9000
+#proactive.pnps.idle_timeout=60000
+#proactive.pnps.keystore=/path/to/keystore.jks
```
Comment (fviale) [inline config/network/server.ini]: same remarks as above

File: config/network/server.ini [hunk 5]
```diff
@@ -1,18 +1,35 @@
...
 #proactive.pamr.router.address=hal
 #proactive.pamr.router.port=33647
 proactive.pamr.agent.magic_cookie=scheduler
 proactive.pamr.agent.id=0
 
 ### Common network configuration
+#proactive.net.interface=ethX
+#proactive.net.netmask=192.168.1.0/24
+
+# Useful when hostname resolution fails
```
Comment (fviale) [inline config/network/server.ini]: duplicates here

File: config/network/node.ini (hunk 1 above)
Comment (lpellegr) [inline config/network/node.ini]: Both keystores are not used for the same purpose.

File: config/network/node.ini (hunk 2 above)
Comment (lpellegr) [inline config/network/node.ini]: `your-password` is given as an example not as the default value.

File: config/network/server.ini (hunk 4 above)
Comment (lpellegr) [inline config/network/server.ini]: Same answer as above.

**Pull request merged**